tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,531 @@
+<tool
+  id="mqppep_preproc"
+  name="MaxQuant Phosphopeptide Preprocessing"
+  version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+  python_template_version="3.5"
```

Comment:
the python_template_version you can remove from all tools

Reply:
okay, I will try it